src/common/string_util.cc

```diff
@@ -312,4 +314,47 @@ std::vector<std::string> TokenizeRedisProtocol(const std::string &value) {
   return tokens;
 }
 
+/* convert string to an escaped representation where
+ * all the non-printable characters (tested with isprint()) are turned into
+ * escapes in the form "\n\r\a...." or "\x<hex-number>". */
+std::string StringRepr(const std::string &s) {
```

Comment:
StringRepr => EscapeString

BTW, can you add the cpp test case for this function?

Reply:
@git-hulk  function name was changed, and test case was added.

Reply:
Thanks